api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -420,17 +420,14 @@ public DbWorkspace saveAndCloneCohortsConceptSetsAndDataSets(DbWorkspace from, D
 
   @Override
   public WorkspaceAccessLevel getWorkspaceAccessLevel(String workspaceNamespace, String workspaceId)
-      throws ServerErrorException {
+      throws IllegalArgumentException {
```

Comment:
Sorry to hold out on the approval again but this is another functional change to our app.

Since it's not throwing a ServerErrorException anymore, the HTTP exception handler will create a generic 500 w/o an error message to the user.

My suggestion to change this to a Java level exception also requires changing the call sites to handle this exception and throw the HTTP level exception (ServerErrorException) in order to keep the application behaving the same way.